docs/tool_migration.md

```diff
@@ -34,7 +34,6 @@ Runs diagnostic commands on a specified VM. The main difference from the HotSpot
 - The `-f` option to read commands from a file.
 - The `Perfcounter.print` option for displaying performance counters for the target VM.
 - Selecting VMs by main class instead of VMID.
```

Comment:
"Selecting VMs by main class instead of VMID" can also be removed, this this is the display name.